packages/opentelemetry-instrumentation-anthropic/tests/test_completion.py

```diff
@@ -528,6 +528,39 @@ async def test_async_anthropic_message_create(exporter, reader):
     assert found_choice_metric is True
     assert found_duration_metric is True
     assert found_exception_metric is True
+    
+# Place the new tests here
+@pytest.mark.vcr
+def test_capture_system_message(exporter):
+    client = Anthropic()
+    system_message = "This is a test system message."
+    
+    # Simulate capturing a system message
+    client.capture_system_message(system_message)
```

Comment:
have you ran this? there's no method like this on Anthropic

Reply:
Proper implementation here: https://github.com/traceloop/openllmetry/pull/2238 :)